src/components/shared/Markdown.tsx

```diff
@@ -1,32 +1,58 @@
-import { omit } from 'lodash'
-import { MarkedOptions, parse, parseInline } from 'marked'
-import * as React from 'react'
-import unindent from '../../lib/util/strings/unindent'
-import NotificationText from './NotificationText'
+import { omit } from "lodash";
+import { type MarkedOptions, parse, parseInline } from "marked";
+import type * as React from "react";
+import unindent from "../../lib/util/strings/unindent";
+import NotificationText from "./NotificationText";
 
 interface IProps extends React.HTMLProps<HTMLDivElement> {
   children?: string | null | false;
   marked: typeof parse | typeof parseInline;
   options?: MarkedOptions;
-  element?: keyof HTMLElementTagNameMap;
 }
 
 export function MarkdownDiv(props: IProps) {
-  if (props.children === undefined || props.children === null || props.children === false) {
-    return null
+  if (
+    props.children === undefined ||
+    props.children === null ||
+    props.children === false
+  ) {
+    return null;
   }
-  if (typeof props.children !== 'string') {
-    return <NotificationText type="negative">Markdown content must be a string.</NotificationText>
+  if (typeof props.children !== "string") {
+    return (
+      <NotificationText type="negative">
+        Markdown content must be a string.
+      </NotificationText>
+    );
   }
-  const Element = props.element || 'div'
+
   return (
-    <Element
-      {...omit(props, 'children', 'marked', 'options', 'element')}
-      dangerouslySetInnerHTML={{ __html: props.marked?.(unindent(props.children), props.options) }}
+    <div
+      {...omit(props, "children", "marked", "options")}
```

Comment:
This included a feature that was similar to `asChild` in Radix, but the feature was unused. As it was difficult to get the typing right, I've removed it for now.